src/components/Volunteers/VolunteerProfile.jsx

```diff
@@ -194,7 +194,7 @@ export default ({ volunteer, refer }) => {
       <ProfileField
         editing={false}
         label="HubSpot ID"
-        value={volunteer.hs_id}
+        value={volunteer.hs_id.toString()}
```

Comment:
some ambassadors have int hs_id, some have string

Reply:
I generally avoid `x.toString()` because it'll die if `x` is null.  Safer to say `'' + x`, which can never throw an error.